CMakeLists.txt

```diff
@@ -36,6 +36,10 @@ include(PACE_AddMex)
 include(PACE_FindMatlab)
 include(horace_FindHDF5)
 include(PACE_Version)
+
+# Make analysis ignore Herbert in case of nesting
```

Comment:
why is this set here rather than with the other sets at the top? I assume this must be set before `PACE_CodeAnalysis` is imported?

Reply:
It needs to be below FindHerbert and above CodeAnalysis.

Reply:
would be worth adding that in the comment to stop someone like me reordering for tidyness